sdk/communication/Azure.Communication.Administration/src/CommunicationIdentityClient.cs

```diff
@@ -29,16 +29,34 @@ public CommunicationIdentityClient(string connectionString, CommunicationIdentit
                   options ?? new CommunicationIdentityClientOptions(),
                   ConnectionString.Parse(AssertNotNull(connectionString, nameof(connectionString))))
         { }
-
+        /// <summary> Initializes a new instance of <see cref="CommunicationIdentityClient"/>.</summary>
+        /// <param name="tokenCredential">Token acquired from the Azure Communication Services to be used by service calls</param>
+        /// <param name="endpoint">Endpoint to hit on the server</param>
+        /// <param name="options">Client option exposing <see cref="ClientOptions.Diagnostics"/>, <see cref="ClientOptions.Retry"/>, <see cref="ClientOptions.Transport"/>, etc.</param>
+        public CommunicationIdentityClient(TokenCredential tokenCredential, string endpoint, CommunicationIdentityClientOptions? options = default)
```

Comment:
I think the standard is to have the endpoint as the first parameter. See this: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/BlobClient.cs#L151
Also I think `endpoint` should be of type `Uri` instead of `string`

Reply:
I moved the parameters around.